base/sim/FairModule.h

```diff
@@ -90,9 +90,14 @@ class FairModule : public TNamed
     /** Finish worker run (used in MT mode only) */
     virtual void FinishWorkerRun() const { ; }
 
-    /**template function to construct geometry. to be used in derived classes.*/
+    /**template function to construct geometry. to be used in derived classes.
+     * The first and the third argument are meaningless, just pass nullptr */
```

Comment:
Can you add a `\deprecated` so it shows up in Doxygen's deprecation list?

Reply:
done